parasoft-findings-teamcity-common/src/main/java/com/parasoft/findings/teamcity/common/ReportParserTypes.java

```diff
@@ -21,13 +21,23 @@
 public class ReportParserTypes {
     private static final String SOATEST_LABEL = "Parasoft SOAtest"; //$NON-NLS-1$
     private static final String ANALYZERS_LABEL = "Parasoft Analyzers"; //$NON-NLS-1$
+    private static final String SA_PMD_LABEL = "PMD"; //$NON-NLS-1$
+    private static final String SA_PMDCPD_LABEL = "PMD Copy/Paste Detector (CPD)"; //$NON-NLS-1$
```

Comment:
Since labels might be visible to users in logger, I suggest to change to some name with "Parasoft" in it.
We can use "Parasoft Code Inspection" and "Parasoft Duplicates" for now. (similar to "Parasoft Warnings" in Jenkins)